apps/site/load_tests/locustfile.py

```diff
@@ -1,6 +1,5 @@
-from locust import HttpLocust, TaskSet, task
-from page import Page
-
+from locust import HttpUser, TaskSet, task
+from linediagram import LineDiagramTaskSet
 
 class TransitNearMe(TaskSet):
```

Comment:
Not related to this commit, so feel free to ignore, but I could see the `TransitNearMe` class moving to its own file.